### ISSUE TITLE
Go to folder not working

### ISSUE THREAD

**Issue by chconnor** [opto/nostalgy-xpi#215]
Apologies if this is already known, but:

v4.0.2 on TB 115.3.1 on Kubuntu 23.04

I have tried mapping various keys to trigger the function "go to folder" (previously "G") but I can't seem to make it work. I have tried F and P before realizing those were possibly claimed by TB... Q did not work, neither did X.

Console log doesn't seem to show anything.

**Comment by Remiol** [comment]
Same here

**Comment by hkclark** [comment]
I'm having this issue as well.  I'm on the Windows 10.  TB 115.3.1 64-bit and Nostalgy++ 4.0.2.  Thank you for the great extension.

**Comment by marcg1** [comment]
Also experiencing the Goto issue on Windows 11, TB 115.3.1, Nostalgy++ 4.0.2.  Goto and Save (which is working) are the two main Nostalgy++ features that I use.

Many thanks for Nostalgy++.  Donated to this critical effort a few weeks go.

**Comment by chconnor** [comment]
I also made this issue re: move to folder not working: https://github.com/opto/nostalgy-xpi/issues/214

And let me echo the thanks for the addon as well; Nostalgy++ is a non-negotiable must-have. I hope everyone has donated to the cause. :-)

**Comment by marcg1** [comment]
Seeing the following consistently when I select Shift+H, which I have bound to GoToFolder with a specific target folder.

<img width="450" alt="image" src="https://github.com/opto/nostalgy-xpi/assets/3654353/210d84df-10d0-43aa-a819-0f12eee86c2d">

**Comment by chconnor** [comment]
> Sometimes it works, sometimes it opens the message in a new tab (without moving it).

I also see this sometimes (the new tab).

**Comment by opto** [comment]
yes, the old folder display does no longer exist. That needs to be totally recoded and is under way.

**Comment by opto** [comment]
Please use the fix in the new release

**Comment by Remiol** [comment]
It works the first time I move an email, but then it doesn't anymore. It opens the email in a new tab.

(Thunderbird 115.3.1. 64-bit - Nostalgy++ 4.0.4.)

**Comment by marcg1** [comment]
Only partially working on Windows 11, TB 115.3.1, Nostalgy++ 4.0.4.  

No response at all in main app or debug console to Shift+G: no change in active folder or prompt to pick a folder.  Shift+G does not confict with any other TB binding as far as I can tell.

I have Shift+H bound to GoToFolder with a specific folder.  That does work now.  Did not work under 4.0.2.  However it does generate the following console complaint.

<img width="674" alt="image" src="https://github.com/opto/nostalgy-xpi/assets/3654353/999e0688-5545-4998-9e14-2aff8ba7af4a">

**Comment by opto** [comment]
G ist captured by TB now. You Need some other Letter ,in German TB, v and r worked for me

**Comment by opto** [comment]
Thanks for the Konsole Log. I tested only as Go, Not as Shift Go, yet

**Comment by marcg1** [comment]
Changing to Shift+F fixed it ... thanks!  It throws the following error, but works correctly.

<img width="678" alt="image" src="https://github.com/opto/nostalgy-xpi/assets/3654353/5da1e80d-cd86-4cba-ac2f-1259aaaff6ce">

**Comment by Olifair** [comment]
> G ist captured by TB now. You Need some other Letter ,in German TB, v and r worked for me

V also works in the French TB :-)

**Comment by Remiol** [comment]
It works the first time I move an email, but then it doesn't anymore. It opens the email in a new tab.

>> I'm using "V" - V doesn't work in my TB.

(Thunderbird 115.3.1. 64-bit - Nostalgy++ 4.0.4.)

**Comment by chconnor** [comment]
So far the new version is working for me. Thanks! (Using "x" on linux.)
